File: app/src/main/java/com/teamopensourcesmartglasses/chatgpt/UI/ChatGptSetupFragment.java
package com.teamopensourcesmartglasses.chatgpt.UI;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.teamopensourcesmartglasses.chatgpt.ChatGptBackend;
import com.teamopensourcesmartglasses.chatgpt.databinding.FragmentChatgptsetupBinding;

public class ChatGptSetupFragment extends Fragment {

    private FragmentChatgptsetupBinding binding;

    @Override
    public View onCreateView(
            LayoutInflater inflater, ViewGroup container,
            Bundle savedInstanceState
    ) {
        binding = FragmentChatgptsetupBinding.inflate(inflater, container, false);
        return binding.getRoot();
    }

    public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
    }

    @Override
    public void onDestroyView() {
        super.onDestroyView();
        binding = null;
    }

}

File: app/src/main/java/com/teamopensourcesmartglasses/chatgpt/ui/ChatGptSetupFragment.kt
package com.teamopensourcesmartglasses.chatgpt.ui

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.teamopensourcesmartglasses.chatgpt.databinding.FragmentChatgptsetupBinding

class ChatGptSetupFragment : Fragment() {
    private var binding: FragmentChatgptsetupBinding? = null
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentChatgptsetupBinding.inflate(inflater, container, false)
        return binding!!.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }

    override fun onDestroyView() {
        super.onDestroyView()
        binding = null
    }
}